sdk/monitor/opentelemetry-exporter/.npmignore

```diff
@@ -0,0 +1,13 @@
+# Ignore everything
```

Comment:
does this file make sense? I think we usually only explicitly include a few things in package.json

Reply:
Since this repo doesn't seem to use `.npmignore`, I'll just delete this and use package.json `"files"`